lightning/src/ln/channel.rs

```diff
@@ -62,6 +62,17 @@ pub struct ChannelValueStat {
 	pub counterparty_dust_limit_msat: u64,
 }
 
+pub struct AvailableBalance {
```

Comment:
maybe AvailableBalances or AvailableBalanceBreakdown or something, because otherwise the naming is confusing.

Reply:
Hmm, its not a breakdown, though, is it? They're separate "available balances" not really a breakdown of a "total available balance"